src/Mvc/Url.php

```diff
@@ -16,7 +16,6 @@
 use Phalcon\Di\AbstractInjectionAware;
 use Phalcon\Mvc\Url\Exception;
 use Phalcon\Mvc\Url\UrlInterface;
-use Phalcon\Parsers\Parser;
 
 use function http_build_query;
 use function is_array;
```

Comment:
Missed a reference here with `replacePaths`

Reply:
This is from Volt, need to check what it is and if it still needed.